masonry_testing/src/lib.rs

```diff
@@ -23,10 +23,19 @@ use masonry_core::core::{Widget, WidgetId};
 ///
 /// Implements helper methods useful for unit testing.
 pub trait TestWidgetExt: Widget + Sized + 'static {
+    // TODO - Remove, keep only record()
```

Comment:
This TODO comment probably need an explanation of how tests which use the one form are migrated/why they haven't been already